### ISSUE TITLE
little bug on 2.3.02 - can't install this payment module from admin (fixed code)

### ISSUE THREAD

**Issue by sunrise-xx** [craigchristenson/opencart-2checkout-api#10]
can't install this payment module from admin:

This is bug , need fix , it miss .tpl

admin/controller/payment/twocheckout_api.php

Line # $this->response->setOutput($this->load->view('payment/twocheckout_api', $data));

change to
$this->response->setOutput($this->load->view('payment/twocheckout_api.tpl', $data));

**Comment by sunrise-xx** [comment]
More bug need fix:

catalog/controller/payment/twocheckout_api.php
Line # 44 need change to 
        return $this->load->view('default/template/payment/twocheckout_api.tpl', $data);

**Comment by craigchristenson** [comment]
We recently updated the module for 2.3 and this issue should be fixed. Please let us know if you are still seeing an error in [the 2.3 branch](https://github.com/craigchristenson/opencart-2checkout-api/tree/v2.3).

**Comment by fashionkartz** [comment]
when i install this plugin i get the errrorr 
Fatal error: Class 'Controllerextensionpaymenttwocheckoutapi' not found in public_html/system/storage/modification/system/engine/action.php on line 43

**Comment by craigchristenson** [comment]
Hi fashionkartz, are seeing this in opencart 2.3x? If so are you using the [v2.3 branch](https://github.com/craigchristenson/opencart-2checkout-api/tree/v2.3)?

**Comment by fashionkartz** [comment]
yes i am using  opencart 2.3.x and also using v2.3 branch .but getting error

**Comment by craigchristenson** [comment]
Thanks for letting me know, looks like some changes were missing. We have updated the v2.3 branch and verified the changes against OC 2.3.0.1.

**Comment by calebh05** [comment]
@fashionkartz Have you had a chance to implement these changes by chance?

**Comment by may1991** [comment]
Hi, i am getting error: "Requested json parse failed" when customer on my site enters incorrect card information and tries to enter card information again.
